config.yaml

```diff
@@ -152,6 +152,21 @@ options:
     description: |
       Use this configuration to extend the HTML sanitization schema.
     type: string
+  global-async-queries:
+    description: |
+      Enable asynchronous querying for charts and dashboard.
+    type: boolean
+    default: False
+  global-async-queries-jwt:
+    description: |
+      jwt for communication with redis database for async querying
+    type: string
+    default: "18b2f8fcd0d708d270c00508da6e8dfc7a21eff14ea438056809805150439a04"
```

Comment:
Where is this value from?

Reply:
It's just a default created with `openssl`